src/node_crypto.cc

```diff
@@ -2544,7 +2549,7 @@ int VerifyCallback(int preverify_ok, X509_STORE_CTX* ctx) {
 static bool IsSupportedAuthenticatedMode(int mode) {
   return mode == EVP_CIPH_CCM_MODE ||
          mode == EVP_CIPH_GCM_MODE ||
-         mode == EVP_CIPH_OCB_MODE;
+         IS_OCB_MODE(mode);
```

Comment:
The same comment as below applies here. `IsSupportedAuthenticatedMode()` is used to check for invalid API use. With this change, when the mode is not implemented by the crypto library, the checking for invalid or missing IV  lengths done by `CipherBase::InitIv()` will be bypassed.